mmdet/core/hook/sync_norm_hook.py

```diff
@@ -0,0 +1,40 @@
+from collections import OrderedDict
+
+from mmcv.runner import get_dist_info
+from mmcv.runner.hooks import HOOKS, Hook
+from torch import nn
+
+from ..utils.dist_utils import all_reduce_dict
+
+
+def get_async_norm_states(module):
+    async_norm_states = OrderedDict()
+    for name, child in module.named_modules():
+        if isinstance(child, nn.modules.batchnorm._NormBase):
+            for k, v in child.state_dict().items():
+                async_norm_states['.'.join([name, k])] = v
+    return async_norm_states
+
+
+@HOOKS.register_module()
+class SyncNormHook(Hook):
+    """Synchronize Norm states after training epoch, currently used in YOLOX.
+
+    Args:
+        interval (int): Synchronizing norm interval. Default to 1.
```

Comment:
`Default to 1.` -> `Default: 1.` We need to keep the docstring convention. Same as all other `Default`.

Reply:
Hi @RangiLyu , Done here as suggested!